lib/gringotts/gateways/pin_payments.ex

```diff
@@ -209,7 +202,7 @@ defmodule Gringotts.Gateways.PinPayments do
     "Basic #{hash}"
   end
 
-  defp extract_card_token({:ok, %{token: token}}) do
+  defp extract_card_token(%{temp_token: token}) do
```

Comment:
I don't understand why this can't be
```elixir
defp extract_card_token(%{id: token}), do: {:ok, token}
```
because `:temp_token` is the same as `:id` in [L229](https://github.com/aviabird/gringotts/pull/134/files/5ab9ae7d227f0ae626d177850093b718bdc5db1f..af09c702b3179090be26118d2ce0aa3e970e8301#diff-ee335c72a2ba6bd364225f8d5bc286deR229)